apps/meteor/app/livechat/server/lib/RoutingManager.js

```diff
@@ -106,7 +106,7 @@ export const RoutingManager = {
 		const user = await Users.findOneById(agent.agentId);
 		const room = await LivechatRooms.findOneById(rid);
 
-		await Message.saveSystemMessage('command', rid, 'connected', user);
+		await Promise.all([Message.saveSystemMessage('command', rid, 'connected', user), Message.saveSystemMessage('uj', rid, '', user)]);
```

Comment:
Sorry I didn't get what you were referring to here Kevin. Are you referring to the user join system message? If so then we had to move it here from `forwardRoomToAgent` method because of the issue mentioned by Harmeet here -> https://rocketchat.atlassian.net/browse/OC-529?focusedCommentId=15118